client/src/utils/attributeAndParamUtils.js

```diff
@@ -22,7 +22,7 @@ const getRobotId = () => JSON.parse(sessionStorage.getItem(ROBOT_ID_PATH));
  * @param {String} robotId the robotId ot the currently opened robot
```

Comment:
I couldn't quite check if everything is right because most of is again is only prettier reformatting. I looked at the methods you added and they seemed okay.